covid19/views/auth/login_view.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+from rest_framework.authentication import BasicAuthentication
+from rest_framework.authtoken.models import Token
+from rest_framework.decorators import permission_classes as perm_classes
+from rest_framework.permissions import AllowAny, IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+logger = logging.getLogger('django')
+
+@perm_classes((AllowAny, ))
+class Login(APIView):
+    authentication_classes = [BasicAuthentication]
+    permission_classes = [IsAuthenticated]
```

Comment:
There are multiple permission configurations in this view.

Reply:
the perm_classes (alias for `permission_classes`) is to allow accessing this API without token authentication, while the `authentication_classes` and `permission_classes` are for basic authentication.